zstacklib/zstacklib/scripts/rbd_rw.py

```
@@ -143,13 +146,14 @@ def listen_pipe():
             if op == 'readhb':
                 offset = int(splits[2])
                 size = int(splits[3])
-                content = read_rbd_image(pool_name, image_name, offset, size, stream=None)
+                content = read_rbd_image(pool_name, image_name, fs_id, offset, size, stream=None)
```

Ruff (0.8.2)

Local variable `fs_id` referenced before assignment

(F823)